.github/workflows/test-dbt-installation-docker.yml

```diff
@@ -119,15 +135,37 @@ jobs:
           image: ${{ steps.image-info.outputs.ref }}
           run: dbt --version
 
-  slack-notification:
-    name: "Post Scheduled Run Failures"
-    needs: docker-installation-test
+      - name: "Dump Job Status"
+        if: ${{ always() }}
+        id: job-status
+        run: |
+          file="test-${{ strategy.job-index }}-tag-${{ matrix.tag }}.json"
+          # Create file
+          touch $file 
+          # Write job status to file
+          echo $JOB_STATUS >> $file
+          # Set path to file for subsequent steps          
+          echo "path=$file" >> $GITHUB_OUTPUT
+        env:
+          JOB_STATUS: ${{ toJson(job.status) }}
+
+      - name: "Upload Job Status For Slack Notification"
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ needs.generate-artifact-name.outputs.artifact-name }}
+          path: ${{ steps.job-status.outputs.path }}
+          retention-days: ${{ env.ARTIFACT_RETENTION_DAYS }}
+
+  send-slack-notification:
+    needs: [generate-artifact-name, docker-installation-test]
     if: ${{ failure() && github.event_name == 'schedule' }}
 
-    uses: dbt-labs/dbt-release/.github/workflows/slack-post-notification.yml@main
+    uses: ./.github/workflows/test-dbt-installation-notify-job-statuses.yml
```

Comment:
```suggestion
    uses: ./.github/workflows/test-dbt-installation-notify-job-statuses.yml@main
```

Reply:
I'm using a local reference for workflows. Such reference type doesn't support any tags and [uses the same commit as the caller workflow](https://github.blog/changelog/2022-01-25-github-actions-reusable-workflows-can-be-referenced-locally/). If it makes sense, I can change the ref type and use the `main` branch as a tag.